### PR TITLE
docs(prs): add pull request template

### PR DESCRIPTION
**Pull request by rubinovitz**
How does this look? Trying to make sure we remember any necessary changes while not impeding contribution.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+## PR Checklist
+
+### Testing
+- [ ] added relevant test coverage
+- [ ] no tests needed
+
+### Docs
+- [ ] added relevant docs
+- [ ] updated relevant sections in the README.md
+- [ ] no docs needed
```
Comment (smothers) [inline PULL_REQUEST_TEMPLATE.md:13]: I think we should add a section for updating the docstrings in the index.d.ts

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## PR Description
```
Comment (smothers) [inline PULL_REQUEST_TEMPLATE.md:1]: Can you rename to `.github/pull_request_template.md`? I don't want to add more config to the root if we can avoid it.

Comment (rubinovitz) [reply PULL_REQUEST_TEMPLATE.md]: ah thanks for catching that! did it!

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (rubinovitz) [reply PULL_REQUEST_TEMPLATE.md]: you got it. done!

**Comment by rubinovitz** [comment]
@smothers I think I resolved your issues. Please re-review when you get a chance.

**Review by smothers — APPROVED** [review]
✅

**Pull request merged**